common/djangoapps/student/models.py

```diff
@@ -743,6 +744,11 @@ def invalidate_user_profile_country_cache(sender, instance, **kwargs):  # pylint
         log.info("Country changed in UserProfile for %s, cache deleted", instance.user_id)
 
 
+@receiver(manage_user_cmd)
+def user_profile_manage(sender, user, **kwargs):  # pylint: disable=unused-argument
+    user_profile, _ = UserProfile.objects.get_or_create(user=user)  # pylint: disable=unused-variable
```

Comment:
instead of disabling the unused-variable pylint warning, just don't assign to a variable.
```suggestion
    UserProfile.objects.get_or_create(user=user)
```

Reply:
This separate signal has been removed and `UserProfile` bit is handled in the `User.post_save` signal.